frontend/src/general/userForm/__tests__/FormConsole.test.jsx

```diff
@@ -0,0 +1,126 @@
+import { describe, expect, it, jest } from "@jest/globals";
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { Button, Container, TextField } from "@material-ui/core";
+import FormConsole from "../FormConsole";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<FormConsole/> class component", () => {
+  const mockHandleSubmit = jest.fn();
+  const dummyValues = {
+    handleSubmit: mockHandleSubmit,
+    buttonName: "Create"
+  };
+
+  const wrapper = Enzyme.mount(
```

Comment:
why mount? why not shallow?

Reply:
That was accidental. I'll change it to shallow, should still work/pass.